### ISSUE TITLE
Review python API of FTRL model

### ISSUE THREAD

**Issue by st-pasha** [h2oai/datatable#1528]
Since `Ftrl` model goes public in the next release, we should take a closer look at its public API. Once it is released, it would be much harder to change in the future.

For example, property `ftrl.fi` can be renamed into `.feature_importance` (since "fi" is too cryptic).

**Comment by oleksiyskononenko** [comment]
Yep, I've been thinking about `fi` and It seems like [some packages](https://rdrr.io/github/Laurae2/Laurae/man/lgbm.fi.html ) indeed use this name. Though may be `varimp` would be better, as it is also kind of [common](https://www.rdocumentation.org/packages/caret/versions/6.0-81/topics/varImp), but has more intuitive name.

**Comment by st-pasha** [comment]
Your links are from R packages. Python is usually more verbose: for example [in scikit-learn](https://scikit-learn.org/stable/modules/feature_selection.html) this is called `.feature_importances_`, and in python `h2o` we use `variable_importances`.

**Comment by oleksiyskononenko** [comment]
Hm, here I see that [Python H2O](http://docs.h2o.ai/h2o/latest-stable/h2o-docs/faq/python.html) uses `varimp`.
